jscomp/stdlib/pervasives.js

```diff
@@ -81,7 +81,11 @@ var $caret = Caml_string.add;
 
 function char_of_int(n) {
   if (n < 0 || n > 255) {
-    return invalid_arg("char_of_int");
+    throw [
+          0,
+          Caml_exceptions.Invalid_argument,
```

Comment:
can we simply use "Invalid_argument" instead of `Caml_exceptions.Invalid_argument`